app/src/main/java/org/mozilla/focus/browser/integration/BrowserToolbarIntegration.kt

```diff
@@ -262,12 +293,38 @@ class BrowserToolbarIntegration(
                 .collect { showTrackingProtectionCfrForTab ->
                     if (showTrackingProtectionCfrForTab[store.state.selectedTabId] == true) {
                         CFRPopup(
-                            container = fragment.requireView(),
-                            text = fragment.getString(R.string.cfr_for_toolbar_shield_icon2),
                             anchor = toolbar.findViewById(
                                 R.id.mozac_browser_toolbar_tracking_protection_indicator,
                             ),
-                            onDismiss = ::onDismissTrackingProtectionCfr,
+                            properties = CFRPopupProperties(
+                                popupWidth = 256.dp,
+                                popupAlignment = CFRPopup.PopupAlignment.INDICATOR_CENTERED_IN_ANCHOR,
+                                popupBodyColors = listOf(
+                                    ContextCompat.getColor(
+                                        fragment.requireContext(),
+                                        R.color.cfr_pop_up_shape_end_color,
+                                    ),
+                                    ContextCompat.getColor(
+                                        fragment.requireContext(),
+                                        R.color.cfr_pop_up_shape_start_color,
+                                    ),
+                                ),
+                                dismissButtonColor = ContextCompat.getColor(
+                                    fragment.requireContext(),
+                                    R.color.cardview_light_background,
+                                ),
+                                popupVerticalOffset = 0.dp,
+                            ),
+                            onDismiss = { onDismissTrackingProtectionCfr() },
+                            text = {
+                                Text(
+                                    fontSize = 16.sp,
+                                    letterSpacing = 0.5.sp,
+                                    lineHeight = 24.sp,
```

Comment:
Seems like this configuration is common to all CFRs.
Could it be extracted in a new typography?

Reply:
Done